ammo.idl

```diff
@@ -504,8 +505,21 @@ btCollisionDispatcher implements btDispatcher;
 interface btOverlappingPairCallback {
 };
 
+
```

Comment:
please remove the extra empty lines (should be just one empty line between interfaces).